lib/audit/duplicateFile.js

```diff
@@ -0,0 +1,31 @@
+const glob = require('tiny-glob');
+const fs = require('fs')
+
+const underScorePrefix = /^_*/;
+
+module.exports = {
+    audit: async () => {
+        let results = {};
+
+        files = await glob('app/**/_*.liquid', { filesOnly: true });
```

Comment:
Probably something like this will be more complete for modules, and i think this `_` thing is only partial thing.
```suggestion
        files = await glob('{app,modules}/**/partials/**/_*.liquid', { filesOnly: true });
```

Reply:
Makes sense, changed it.